eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -54,6 +54,17 @@
       "RunAggregateReports": false,
       "TestGoals": "surefire:test",
       "TestOptions": ""
+    },
+    {
+      "Agent": {
+        "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" }
+      },
+      "JavaTestVersion": "1.17",
+      "AZURE_TEST_HTTP_CLIENTS": "jdk",
+      "TestFromSource": false,
+      "RunAggregateReports": false,
+      "TestGoals": "surefire:test",
+      "TestOptions": ""
```

Comment:
For now, let's leave this change out and just add it into the matrix AppConfiguration is using, this way if the change has issues we don't break all CI runs and only AppConfiguration which we can react to quickly.